tests/test_worker.py

```diff
@@ -1362,3 +1362,56 @@ def setUp(self):
     def test_handle_exception_handles_non_ascii_in_exception_message(self):
         """worker.handle_exception doesn't crash on non-ascii in exception message."""
         self.worker.handle_exception(Mock(), *self.exc_info)
+
+
+class TestRoundRobinWorker(RQTestCase):
+    def test_round_robin(self):
+        qs = [Queue('q%d' % i) for i in range(5)]
+
+        for i in range(5):
+            for j in range(3):
+                qs[i].enqueue(say_pid,
+                              job_id='q%d_%d' % (i,j))
+
+        w = RoundRobinWorker(qs)
+        w.work(burst=True)
+        start_times = []
+        for i in range(5):
+            for j in range(3):
+                job = Job.fetch('q%d_%d' % (i,j))
+                start_times.append(('q%d_%d' % (i,j), job.started_at))
+        sorted_by_time = sorted(start_times, key=lambda tup: tup[1])
+        sorted_ids = [tup[0] for tup in sorted_by_time]
+        expected = ['q%d_%d' % (i,j) for j in range(3) for i in range(5)]
+        self.assertEqual(expected, sorted_ids)
+
+
+class TestRandomWorker(RQTestCase):
+    def test_random_worker(self):
+        qs = [Queue('q%d' % i) for i in range(5)]
+
+        for i in range(5):
+            for j in range(3):
+                qs[i].enqueue(say_pid,
+                              job_id='q%d_%d' % (i,j))
```

Comment:
Can you add space after `,`? Sorry for nitpicking, but I want the code style to be consistent.

Reply:
done